src/components/TeamProjectCard.vue

```diff
@@ -3,14 +3,14 @@
     <div
       class="project-image"
       :class="{
-        'name-on-hover': !team.project.images.bannerHasName
+        'name-on-hover': !team.project.banner.hasName
       }"
       :style="{
-        'background-image': 'url(' + team.project.images.bannerURI + ')',
+        'background-image': 'url(' + team.project.banner.url + ')',
```

Comment:
nit: can we use template strings here i.e. `some ${variable}`

Reply:
you cannot use es6 template strings inside vue bindings AFAIK: https://github.com/vuejs/vue/issues/2364